### PR TITLE
Add firing buffer to sticky launcher

### PR DESCRIPTION
**Pull request by benjl**
Currently, consistently firing the stickybomb launcher is very precise, as releasing primary fire even a frame too early causes the weapon to not fire.  
This addresses that by adding a small buffer zone. If primary fire is released up to 0.2 seconds before you can fire, the weapon is fired as soon as possible.  
You can play around with mom_sj_buffer_window in mapping mode to test out different window lengths.

### Checklist
- [x] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [x] My commits are relatively small and scoped to the best of my ability
- [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

**Comment by TheRealHona** [comment]
Does this decrease the skill required to play however?

**Comment by benjl** [comment]
IMO it shouldn't meaningfully impact the skill required to play, just eliminate the occasional missed input from firing a bit too early. The window could be made a lot smaller though if 0.2 seconds feels too forgiving.

File: mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp [hunk 1]
```diff
@@ -19,6 +19,8 @@
 #define MOM_STICKYBOMB_MAX_CHARGE_VEL 2400
 #define MOM_STICKYBOMB_MAX_CHARGE_TIME 4.0f
 
+#define MOM_STICKYLAUNCHER_BUFFER_WINDOW 0.2f
```
Comment (TheRealHona) [inline mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp]: Should this be MOM_STICKYLAUNCHER_DET_BUFFER_WINDOW instead to be more specific?

File: mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp [hunk 2]
```diff
@@ -65,6 +65,8 @@ static MAKE_TOGGLE_CONVAR_CV(mom_sj_charge_enable, "1", FCVAR_ARCHIVE,
 );
 #endif
 
+static ConVar mom_sj_buffer_window("mom_sj_buffer_window", "0.2", FCVAR_MAPPING, "Amount of time in seconds you can buffer a primary fire before you can shoot.", true, 0, true, 0.6f);
```
Comment (TheRealHona) [inline mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp]: Should this be mom_sj_det_buffer_window (or similar) instead to be more specific?

**Pull request merged**